### ISSUE TITLE
2FA request

### ISSUE THREAD

**Issue by PedroCisnerosSantana** [appwrite/appwrite#249]
Hello, I'd like to work on https://github.com/appwrite/appwrite/projects/1#card-25704781 :smile: 
what lang can I use?

**Comment by PedroCisnerosSantana** [comment]
@eldadfux

**Comment by eldadfux** [comment]
We could definitely use help on this but we still need to write a proper spec before we can move forward. If you would like to help me, you can reach me on Discord.

**Comment by eldadfux** [comment]
Closing in favour of #244
